components/service-item/service-item.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Link from 'next/link'
+import { ButtonSecondaryMock } from '..'
+
+const ServiceItem = ({ item = {}, Component = null, onMouseOver = null, onMouseLeave = null }) => {
+  const hasLink = item.link && item.link.slug
+
+  return hasLink
+    ? <WithLink {...item} Component={Component} onMouseOver={onMouseOver} onMouseLeave={onMouseLeave} />
+    : <Item {...item} Component={Component} />
+}
+
+ServiceItem.propTypes = {
+  item: PropTypes.object,
+  Component: PropTypes.func,
+  onMouseOver: PropTypes.func,
+  onMouseLeave: PropTypes.func,
+}
+
+const WithLink = ({
+  button = '',
+  iconColor = '',
+  link = '',
+  text = '',
+  title = '',
+  Component = null,
+  onMouseOver = null,
+  onMouseLeave = null,
+}) => (
+  <Link href={`/service?slug=${link.slug}`} as={`/service/${link.slug}`}>
+    <a
```

Comment:
A screen reader would now read the whole block as the link text. This can be solved by wrapping only the `h3` in an `a` tag and make the whole block clickable using an `:after` (or some JS of you'd prefer that). You can see an example on this technique on the post blocks on [smashingmagazine.com](https://www.smashingmagazine.com/).